pom.xml

```diff
@@ -458,6 +459,16 @@
                     <artifactId>build-helper-maven-plugin</artifactId>
                     <version>3.4.0</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.sonarsource.scanner.maven</groupId>
```

Comment:
Do you think sonar should be in a separate profile?  I'm thinking about people who might wish to build Kroxy themselves in a CI system (derivates builds), but wouldn't necessarily want to hook up to Sonar.

Reply:
I don't think the CI profile represents anything other than our CI usage so no I don't. I'll happily shuffle things around if and when someone asks us too.